plugins/include/amxmodx.inc

```diff
@@ -159,6 +159,16 @@ forward client_connect(id);
  */
 forward client_connectex(id, const name[], const ip[], reason[128]);
 
+/**
+ * Called after the client's admin flags are changed.
+ *
+ * @param id        Client index
+ * @param flags     New client flags
+ *
+ * @noreturn
+ */
+forward client_flags_updated(id, flags);
```

Comment:
I'd also pass id of set here, so we'll know which set of flags has been updated.

Reply:
I never knew different sets of flag existed until I saw the actual .cpp file. Are different sets ever used? Because if not, it would be rather unnecessary to have to write the set argument every time we use the forward.

Reply:
We don't have to write all params in forward, they can be skipped, so if someone doesn't depend on sets, they can safely omit last param.

Reply:
I knew they existed but I never knew their purpose. The doc says:
> AMXX stores multiple sets of flags internally, but only flag set
0 is actively used. You should not change the value of the third
parameter from the default.

I almost feel that we should just ignore flag sets and use a single one.

Reply:
Even if AMXX doesn't use other values than 0, it still may be used by 3rd plugins for their some specific permission system, so they don't mess up with the AMXX's one. I think including that param wouldn't do much harm, but it's just my opinion.

Reply:
@Amaroq7 we can add it later if someone really uses sets.

Reply:
Actually if they can really be skipped when using the forwards (haven't actually checked this), then there's really no problem if that param can be passed too. I'll check this tomorrow and add it if that's the case.

Reply:
@Amaroq7 @rsKliPPy @WPMGPRoSToTeMa - I added the flag set as the last parameter in the forward, so people can skip it if not needed. I also passed the old flags so people can see which flags were changed.